pkg/ui/src/util/events.ts

```diff
@@ -27,14 +27,15 @@ export function getEventDescription(e: Event$Properties): string {
     case eventTypes.CREATE_DATABASE:
       return `Database Created: User ${info.User} created database ${info.DatabaseName}`;
     case eventTypes.DROP_DATABASE:
-      info.DroppedTables = info.DroppedTables || [];
-      let tableDropText: string = `${info.DroppedTables.length} tables were dropped: ${info.DroppedTables.join(", ")}`;
-      if (info.DroppedTables.length === 0) {
-        tableDropText = "No tables were dropped.";
-      } else if (info.DroppedTables.length === 1) {
-        tableDropText = `1 table was dropped: ${info.DroppedTables[0]}`;
+      info.DroppedSchemaObjects = info.DroppedSchemaObjects || [];
```

Comment:
I find this defensiveness suspicious.  If we don't have an array here at all, doesn't that mean we just don't have the information?  I'd be inclined to just not include the second sentence in that case.

Reply:
Done.